modules/azure-gaming/game-dev-vmss/README.md

```diff
@@ -0,0 +1,67 @@
+# Azure GameDev VMSS
+
+Bicep Module to simplify deployment of the Azure Game Developer VMSS
+
+## Description
+
+{{ Add detailed description for the module. }}
```

Comment:
Could you add detailed description for the module?

Reply:
```suggestion
Enable the deployment of the Azure Game Developer Virtual Machine Images onto Azure VMSS for use in Game Development Scenarios.
```